AutoRest/TestServer/swagger/httpInfrastructure.json

```diff
@@ -2795,6 +2795,9 @@
       }
     },
     "A": {
+      "x-ms-exception": {
```

Comment:
In batch scenario, we have error type named: BatchError, but we would like to make the exception named: BatchException instead of BatchErrorException. This extension is optional.